crm_timesheet/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Antiun - Antonio Espinosa
+# Copyright 2015 Antiun - Javier Iniesta
+# Copyright 2017 Tecnativa - David Vidal
+# Copyright 2019 Tecnativa - Jairo Llopis
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "CRM Timesheet",
+    "category": "Customer Relationship Management",
+    "version": "17.0.1.0.0",
+    "depends": ["crm", "project_timesheet_time_control", "sale_timesheet"],
```

Comment:
Why is the sale_timesheet dependency necessary in this version? Thank you

Reply:
Without this dependency, the tests fail because they do not find the billing_type field.

Reply:
I've just tried locally and you don't need that dependency. You have added the billing_type field when creating project1 in test and I don't think that you should have done that (note that the field is not mentioned in v16 test file).

Reply:
Deleted